arch/arm/src/stm32h7/Kconfig

```diff
@@ -293,6 +293,10 @@ config STM32H7_PWR_DIRECT_SMPS_SUPPLY
 	depends on STM32H7_HAVE_SMPS
 	default n
 
+config STM32H7_PWR_EXTERNAL_SOURCE_SUPPLY
+	bool "Use external source as power supply"
+	default n
+
```

Comment:
Please add help 'Select this option for using an external power supply connected directly to the VCAP pin.'

Reply:
@davids5 No problem, it's now added.